app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart

```diff
@@ -49,7 +49,8 @@ class BatchBackfiller extends RequestHandler {
 
   Future<void> backfillRepository(RepositorySlug slug) async {
     final DatastoreService datastore = datastoreProvider(config.db);
-    final List<FullTask> tasks = await (datastore.queryRecentTasks(slug: slug)).toList();
+    final List<FullTask> tasks =
+        await (datastore.queryRecentTasks(slug: slug, commitLimit: config.backfillerCommitLimit)).toList();
```

Comment:
I see that you are passing the limit in so was the limit hardcoded before this change?